test/e2e/common/node/pod_resize.go

```diff
@@ -968,6 +968,322 @@ func doPodResizeTests() {
 				},
 			},
 		},
+		{
+			name: "Guaranteed QoS pod, one restartable init container - increase CPU & memory",
+			containers: []e2epod.ResizableContainerInfo{
+				{
+					Name:      "c1",
```

Comment:
Can you modify the resources as per my changes in https://github.com/kubernetes/kubernetes/pull/128719/files ?